Tests/UITests/TestApp/OnboardingTestsView.swift

```diff
@@ -15,42 +15,41 @@ struct OnboardingTestsView: View {
     @Binding var onboardingFlowComplete: Bool
     @State var showConditionalView = false
 
-
     var body: some View {
-        OnboardingStack(onboardingFlowComplete: $onboardingFlowComplete) {
+        OnboardingStack(didComplete: $onboardingFlowComplete) {
             OnboardingStartTestView(
                 showConditionalView: $showConditionalView
             )
             OnboardingWelcomeTestView()
             OnboardingSequentialTestView()
-
+            
```

Comment:
Nit: Do we really need these diffs? Actually not sure what our Spezi guideline is here.

Reply:
update: i also don't know what the guideline is (or if we even have one), but since all of the other empty lines in the view bofy here are indented, one could argue that also having this one be indented would be more correct than having this one be the odd one out

Reply:
I think we have a guideline somewhere, but it's not a huge deal, just struck me weird to have these diffs in there